.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: java
 jdk:
   - oraclejdk8
-script: mvn clean verify
\ No newline at end of file
+script: mvn clean verify
+sudo: required
```

Comment:
yes to use standard images 

Please refer https://blog.travis-ci.com/2017-05-04-precise-image-updates for more information

![Screen Shot from travis build](https://user-images.githubusercontent.com/24761580/27782601-c2cde622-5ff0-11e7-8e60-b0df60c425a7.png)